x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/TransportDownsampleAction.java

```diff
@@ -465,6 +476,9 @@ public void onResponse(PersistentTasksCustomMetadata.PersistentTask<PersistentTa
                 @Override
                 public void onFailure(Exception e) {
                     logger.error("error while waiting for downsampling persistent task", e);
+                    if (errorReported.getAndSet(true) == false) {
+                        downsampleMetrics.recordLatencyMaster(getDurationInMillis(), DownsampleMetrics.ShardActionStatus.FAILED);
```

Comment:
nit: `ShardActionStatus` is no longer shard-specific, should it be called `ActionStatus`?